server/go.mod

```diff
@@ -149,6 +149,7 @@ require (
 	github.com/smartystreets/assertions v1.1.1 // indirect
 	github.com/smartystreets/goconvey v1.6.4 // indirect
 	github.com/sosodev/duration v1.3.1 // indirect
+	github.com/stretchr/objx v0.5.2 // indirect
```

Comment:
[memo]
I don't add any package but we have ci problem withou this package.
I think someone forget commit this go mod change.